src/main/java/io/cdap/plugin/ariba/source/connector/AribaConnectorConfig.java

```diff
@@ -151,6 +151,7 @@ public void validateCredentials(FailureCollector failureCollector) {
       String errMsg = ResourceConstants.ERR_MISSING_PARAM_PREFIX.getMsgForKey(ResourceConstants.TOKEN_URL);
       failureCollector.addFailure(errMsg, COMMON_ACTION).withConfigProperty(TOKEN_URL);
     }
+    failureCollector.getOrThrowException();
```

Comment:
Ideally this should be added in the caller method which provided the failureCollector instance.

Reply:
Done